workspace-manager-clienttests/src/main/java/scripts/utils/WorkspaceManagerServiceUtils.java

```diff
@@ -0,0 +1,92 @@
+package scripts.utils;
+
+import bio.terra.testrunner.common.utils.AuthenticationUtils;
+import bio.terra.testrunner.runner.config.ServerSpecification;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import bio.terra.workspace.client.ApiClient;
+import com.google.auth.oauth2.AccessToken;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.common.base.Strings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+public class WorkspaceManagerServiceUtils {
+    private static final Logger logger = LoggerFactory.getLogger(WorkspaceManagerServiceUtils.class);
+
+    private WorkspaceManagerServiceUtils() {}
+
+    /**
+     * Build the Workspace Manager Service API client object for the given server specification.
+     *
+     * @param server the server we are testing against
+     * @return the API client object
+     */
+    public static ApiClient getClient(ServerSpecification server) throws IOException {
```

Comment:
Perhaps we can rename this to getClientForTestRunnerSA instead and update the comment to include that it's setting the access token for the Test Runner SA specified for the given server?

Reply:
Refactored build client code.